lib/consumer.js

```diff
@@ -112,7 +112,7 @@ Consumer.prototype.init = function () {
             return self.emit('error', err);
         }
 
-        if (self.options.fromOffset)
+        if (!self.options.fromOffset)
```

Comment:
I think this should not be negated. If `fromOffset` is true, then provided offsets should be used. When you negate this condition, block will continue and fetch offsets from server.